src/locals/hu.json

```diff
@@ -0,0 +1,145 @@
+            "line2": "az automatikus gondolataidat, akkor amikor történnek."
+        },
+        "bad_thought_note": {
+            "line1": "Jegyzeteld le, pl:",
+            "line2": "Nem mentem el Gyuri bulijára."
+        },
+        "identify_distortions": {
+            "line1": "Keress",
+            "emphasis1": "kognitív torzítást"
+        },
+        "bad_distortions": {
+            "line1": "Biztosan tudjuk, hogy Gyuri utál minket? Ha nem, akkor",
+            "emphasis1": " Gondolatot olvasunk",
+            "line2": "Elfelejtettrm Gyuri buliját, biztos utál engem."
+        },
+        "challenge_screen": {
+            "line1": "Miért nem racionális ez a gondolat?",
+            "emphasis1": "Szállj harcba vele!"
+        },
+        "write_challenge": {
+            "line1": "Valami ilyesmit írhatnánk:"
+        },
+        "alt_thought": {
+            "line1": "Végül írunk egy",
+            "emphasis1": "Helyettesítő gondolatot."
+        },
+        "show_off": {
+            "line1": "Találd meg később is:"
+        },
+        "got_it": {
+            "line1": "Tényleg csak ennyi",
+            "btn_title": "Uccu neki!"
+        }
+    },
+    "auto_thought": "Automatikus gondolat",
+    "cog_distortion": "Kognitív Torzítás",
+    "challenge": "Challenge",
+    "alt_thought": "Helyettesítő gondolat",
+    "cbt_form": {
+        "edit": "Szerkeszt",
+        "new": "Új",
+        "save": "Ment",
+        "auto_thought_placeholder": "Mi történik most a fejedben?",
+        "cog_distortion": "Kognitív torzítás",
+        "challenged": "Vitat",
+        "changed_placeholder": "Vitasd meg a gondolatot!",
+        "alt_thought_placeholder": "Mire gondolhatnánk helyette?"
+    },
+    "all_or_nothing_thinking": "Mindent vagy semmit gondolkogás.",
+    "all_or_nothing_thinking_explanation": "Ez a torzítás akkor történik, amikor nem tudunk árnyalatokban gondolkodni. Ha egy kis hibát fedezünk fel magunkban és emiatt az gondoljuk velejünkig romlottak vagyunk, akkor a végletekben való gondolkozás torzítását éljük át.",
+    "all_or_nothing_thinking_thought": "Elbaltáztam az interjúmat, sehova se fognak felvenni.",
+    "catastrophizing": "Katasztrófizálunk",
+    "catastrophizing_explanation": "Ha a bolhából is elefántots csinálunk, akkor katasztofizálunk. Ha egy kis hibát ejtettél a munkahelyeden, még nem biztos, hogy csak a miatt kirúgnak.",
+    "catastrophizing_thought": "Ideges vagyok, biztos szívrohamom van.",
+    "emotional_reasoning": "Érzelmi érvelés",
+    "emotional_reasoning_explanation_1": "\"Érzem, tehát biztos igaz is.\" Ha jogosnak érzed a \"veszélyt\" azért, mert te félsz tőle, akkor elképzelhető, hogy érzelmi érvelést alkalmazol. A dolgok nem azért veszélyesek, mert mi félünk tőle vagy azt gondoljuk, hogy rossz.",
+    "emotional_reasoning_explanation_2": "Ezt sokszor nehéz érzékelni. Nehéz összeszedned magad annyira, hogy logikusan tudj gondolkozni az érzelmeidről is.",
```

Comment:
felismerni / észrevenni?

Reply:
I would like to use "érzékelni" here because it is a more general term.